fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -530,4 +611,24 @@ struct EncryptedMutationsAndVersionRef {
 	};
 };
 
+TEST_CASE("noSim/CommitTransaction/MutationRef") {
```

Comment:
Can you move this to a .cpp file to avoid repeated compilations?

Reply:
Do you mean the compilation when the test is changed? If so, shall we make this change in another PR, there are many method definitions in this .h file, also making the compilation slow.

Reply:
I mean this test case is defined in .h file, which will be included in many .cpp files. As a result, it will be compiled many times. Moving it will reduce compilation time.

Yes, we can do it in another PR.